janusgraph-driver/src/main/java/org/janusgraph/graphdb/tinkerpop/io/binary/GeoshapeGraphBinarySerializer.java

```diff
@@ -40,6 +40,28 @@ public BufferInputStream(Buffer buffer) {
         public int read() {
             return buffer.readInt();
         }
+
+        @Override
+        public int read(byte b[], int off, int len) {
```

Comment:
Code style linter doesn't allow braces to be at C style position. Thus, all tests fail.
To fix that you need to replace `byte b[]` here with `byte[] b`.